modules/ossm-config-openshift-monitoring-ambient-mode.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-ambient-mode_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh ambient mode
+
+[role="_abstract"]
+You can integrate {SMProductName} with user-workload monitoring to enable observability in your service mesh ambient mode. User-workload monitoring provides access to essential built-in tools and is required to run Kiali, the dedicated console for {istio}. 
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You have enabled the user-workload monitoring.
++
+[NOTE]
+====
+You can enable user-workload monitoring by applying the `ConfigMap` change for metrics integration. For more information, see link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+====
+* You have deployed the Bookinfo application in ambient mode to use the following example. For more information, see https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.2/html/installing/ossm-istio-ambient-mode#ossm-deploying-bookinfo-application-istio-ambient-mode_ossm-istio-ambient-mode[Deploying the Bookinfo application in Istio ambient mode].
+
+.Procedure
+
+. Create a YAML file named `service.yml.yml` to define a `ServiceMonitor` resource that uses the metrics exposed by the ztunnel, similar to the following example:
+
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: ztunnel
+  namespace: ztunnel
+  labels:
+    app: ztunnel
+    service: ztunnel
+spec:
+  selector:
+    app: ztunnel
+  ports:
+    - name: http-monitoring
+      protocol: TCP
+      port: 15020
+      targetPort: 15020
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f service.yml
+----
+
+. Create a YAML file named `servicemonitor.yml` to define a `ServiceMonitor` resource that monitors the {istio} control plane, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f servicemonitor.yml
+----
```

Comment:
The "Create a YAML file named..." and Apply the YAML file parts are not consistent with how we describe about creating yaml resources in other docs such as the tracing PR.

We can keep "similar to the following example:" above and remove the "Create a file named ..." and remove the "Apply the YAML file" parts.